lib/manager/gomod/artifacts.ts

```diff
@@ -68,6 +69,20 @@ function getUpdateImportPathCmds(
   return updateImportCommands;
 }
 
+function useModcacherw(goVersion: string): boolean {
+  if (!is.string(goVersion)) {
+    return false;
+  }
+
+  const [majorPart, minorPart] = goVersion.replace(/^[^\d]*/g, '').split('.');
+  const [major, minor] = [majorPart, minorPart].map((x) => parseInt(x, 10));
+  if (Number.isNaN(major) || Number.isNaN(minor)) {
+    return false;
+  }
+
+  return major > 1 || (major === 1 && minor >= 14);
```

Comment:
Can you make sure it would work if one day they specify the constraint using `major.minor.patch`?

Reply:
I've done double-check, it just ignores patch